battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java

```diff
@@ -53,7 +53,8 @@ private Parent createContent() {
         enemyBoard = new Board(true, output);
```

Comment:
true and folse in constructors are like magic number for me, i dont know what they mean

Reply:
good point